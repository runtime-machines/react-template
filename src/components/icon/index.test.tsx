import * as React from 'react';
import { shallow } from 'enzyme';

import { Icon } from '.';

describe('Icon component', () => {
	it('should render successfully', () => {
		const tree = shallow(<Icon src="@assets/logo.svg" />);

		expect(tree).toMatchSnapshot();
	});
});
