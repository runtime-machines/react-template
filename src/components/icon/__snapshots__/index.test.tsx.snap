// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should render successfully 1`] = `
<InlineSVG
  cacheRequests={true}
  className="c-svg-icon"
  src="@assets/logo.svg"
  uniquifyIDs={false}
/>
`;
